Specs/Number/Arabic/NumberRangeModel.json

```diff
@@ -149,18 +148,18 @@
     ]
   },
   {
-    "Input": "على الأكثر ١٠٠ وعلى الأقل ٢٠",
+    "Input": "الرقم ما بين ٢٠ و١٠٠.",
```

Comment:
Please always add a comment in the PR description for such spec edits.

Reply:
Added in PR Description.